### PR TITLE
Added missing async tags

### PR DESCRIPTION
**Pull request by justetz**

### CODE REVIEW

File: src/Actions.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Poll } from "./Poll";
 import { WebClient } from "@slack/web-api";
 import { KnownBlock } from "@slack/types";
-import { Request, Response } from "@types/express";
+// import { Request, Response } from "@types/express";
```
Comment (lramos15) [inline src/Actions.ts]: If you're going to comment it out you might as well delete it.

**Review by lramos15 — CHANGES_REQUESTED** [review]

File: src/Actions.ts [hunk 2]
```diff
@@ -108,7 +108,7 @@ export class Actions {
         }
     }
     
-    private onDeleteSelected(payload, poll: Poll): void {
+    private async onDeleteSelected(payload, poll: Poll): void {
```
Comment (lramos15) [inline src/Actions.ts]: These return types now should be Promise<Void>

**Review by lramos15 — CHANGES_REQUESTED** [review]

File: src/Actions.ts (hunk 2 above)
Comment (justetz) [reply src/Actions.ts]: Done

File: src/Actions.ts [hunk 3]
```diff
@@ -74,7 +75,7 @@ export class Actions {
         }
     }
     
-    private onResetSelected(payload, poll: Poll): void {
+    private async onResetSelected(payload, poll: Poll): Promise<void> {
```
Comment (lramos15) [inline src/Actions.ts]: No implicit any. This won't compile

**Review by lramos15 — CHANGES_REQUESTED** [review]
Please fix code climate issue as well :)

**Review by lramos15 — APPROVED** [review]
Thanks for the fixes 👍

**Pull request merged**